pkg/cmd/cli/cmd/project.go

```diff
@@ -168,13 +169,25 @@ func (o ProjectOptions) RunProject() error {
 		project, err := o.Client.Projects().Get(argument)
 		if err != nil {
 			if isNotFound, isForbidden := kapierrors.IsNotFound(err), clientcmd.IsForbidden(err); isNotFound || isForbidden {
-				msg := fmt.Sprintf("A project named %q does not exist or you do not have rights to view project on server %q.", argument, clientCfg.Host)
+				var msg string
+				if isForbidden {
+					msg = fmt.Sprintf("You are not a member of project %q.", argument)
+				} else {
+					msg = fmt.Sprintf("A project named %q does not exist on %q.", argument, clientCfg.Host)
+				}
 
 				projects, err := getProjects(o.Client)
 				if err == nil {
-					msg += "\nYour projects are:"
-					for _, project := range projects {
-						msg += "\n" + project.Name
+					switch len(projects) {
+					case 0:
+						msg += "\nYou are not a member of any projects. You can request a project to be created with the 'new-project' command."
```

Comment:
Do we have the correct command prefix passed in?  I thought we didn't, so 'osc' isn't safe to hardcode.

----- Original Message -----
> >  
> >  				projects, err := getProjects(o.Client)
> >  				if err == nil {
> > -					msg += "\nYour projects are:"
> > -					for _, project := range projects {
> > -						msg += "\n" + project.Name
> > +					switch len(projects) {
> > +					case 0:
> > +						msg += "\nYou are not a member of any projects. You can request a
> > project to be created with the 'new-project' command."
> 
> `osc new-project` ?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2405/files#r30793999